### PR TITLE
Change locale for Danish/Denmark to da_DK so it matches ISO-3166 and …

### PR DESCRIPTION
**Pull request by MartinMSPedersen**
The correct locale for danish in Denmark is da_DK as specified in ISO-3166 and ISO-369-1.

### CODE REVIEW

**Comment by sckott** [comment]
thanks very much @MartinMSPedersen having a look now.

File: R/phone-number-provider-all.R [hunk 1]
```diff
@@ -114,19 +114,16 @@ phone_number_formats_el_gr = c(
...
-  '+45(0) #########',
-  '+45(0)#### #####',
-  '0##########',
-  '0#########',
-  '0#### ######',
-  '0#### #####',
-  '(0####) ######',
-  '(0####) #####'
-)
+phone_number_formats_da_dk = c(
```
Comment (sckott) [inline R/phone-number-provider-all.R]: @MartinMSPedersen  You removed some of the possible formats. curious if you just now the possible formats, or if you have a reference on the web?

Comment (MartinMSPedersen) [reply R/phone-number-provider-all.R]: Phonenumbers in denmark is always 8 digits.
The country-code is 45 and it is something prefixed.
And there is never a zero between the country-code and the number.

Regards
Martin

Comment (MartinMSPedersen) [reply R/phone-number-provider-all.R]: Thanks for the package. Yes, I know the danish number. I am danish and have lived all my life in Denmark.
You can also check this page:
https://en.wikipedia.org/wiki/Telephone_numbers_in_Denmark

Regards
Martin

Comment (sckott) [reply R/phone-number-provider-all.R]: Okay. Thanks very much.

**Pull request merged**